core/trino-main/src/test/java/io/trino/sql/gen/TestExpressionCompiler.java

```diff
@@ -1476,6 +1484,26 @@ public void testFunctionCallRegexp()
         Futures.allAsList(futures).get();
     }
 
+    @Test
+    public void testSearchInterruptible()
```

Comment:
This test looks identical to the test in `TestLikeFunctions`. What's the difference?

Also it doesn't feel like these tests belong in those clases, as technically they don't validate expressions but instead (please correct me if I'm wrong) make direct calls to a specific library (JONI). I would recommend extracting this test into it's own test class.

Reply:
We should extract it into a separate test class. It doesn't seem like these tests should live in `TestLikeFunctions` / `TestExpressionCompiler`